back/src/forms/database.ts

```diff
@@ -11,20 +11,27 @@ import { FullForm } from "./types";
 
 /**
  * Returns a prisma Form with all linked objects
- * (owner, ecoOrganisme, temporaryStorage, transportSegments)
+ * (owner, ecoOrganisme, temporaryStorage, transportSegments, intermediaries)
  * @param form
  */
 export async function getFullForm(form: Form): Promise<FullForm> {
   const temporaryStorageDetail = await prisma.form
-    .findUnique({ where: { id: form.id } })
+    .findUnique({
+      where: { id: form.id }
+    })
     .temporaryStorageDetail();
-  const transportSegments = await prisma.form
-    .findUnique({ where: { id: form.id } })
+  const transportSegments = await await prisma.form
+    .findUnique({
+      where: { id: form.id }
+    })
     .transportSegments();
   return {
     ...form,
     temporaryStorageDetail,
-    transportSegments
+    transportSegments,
+    intermediaries: await prisma.intermediaryFormAssociation.findMany({
```

Comment:
Pour profiter des dataloaders natifs Prisma, il vaudrait p-e mieux faire ici:
```suggestion
    intermediaries: await prisma.form
    .findUnique({
      where: { id: form.id }
    }).intermediarries()
```